pkg/services/ngalert/api/compat.go

```diff
@@ -186,7 +186,7 @@ func AlertRuleExportFromAlertRule(rule models.AlertRule) (definitions.AlertRuleE
 		IsPaused:     rule.IsPaused,
 	}
 	if rule.For.Seconds() > 0 {
-		result.ForSeconds = util.Pointer(int64(rule.For.Seconds()))
+		result.ForString = util.Pointer(model.Duration(rule.For).String())
```

Comment:
Why we need to turn a `time.Duration` into `model.Duration` before encoding it as a string? I'm not very familiar this with code, so there might be a good reason for it that I'm just not seeing.

Reply:
I don't have a good answer other than we use the Prometheus Duration parser in other places of the API (and the [Terraform Provider](https://github.com/grafana/terraform-provider-grafana/blob/494629dd0aeea2b6a9602da6b1bc87206ef4daa3/internal/common/schema.go#L70)), so I kept to the same type for consistency.

Reply:
Looks like it's quite different from `time.Duration` (https://github.com/prometheus/common/blob/main/model/time.go#L204) 😄